gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/FlowSpecSearchObject.java

```diff
@@ -116,6 +119,13 @@ public String augmentBaseGetStatement(String baseStatement)
       conditions.add("owning_group = ?");
     }
 
+    if (this.getCount() > 0) {
```

Comment:
Out of curiosity, do we/restli guard against count == 0 or negative values?

Reply:
I don't think restli guards against count == 0 or negative values
`In order for the Rest.li framework to automatically construct Link objects, certain conditions must be met. For both previous and next links, the count in the request must be greater than 0. For links to the previous page, start must be greater than 0.` from the restli server documentation https://linkedin.github.io/rest.li/user_guide/restli_server

We are guarding against it on the Gobblin side by doing the check for inserting the `LIMIT` and `OFFSET` only when the count and start inputted by the user is > 0